monai/networks/nets/classifier.py

```diff
@@ -24,18 +25,18 @@ class Classifier(Regressor):
 
     def __init__(
         self,
-        in_shape,
-        classes,
-        channels,
-        strides,
-        kernel_size=3,
-        num_res_units=2,
+        in_shape: Tuple[int, ...],
```

Comment:
I am a little bit confusing that: now we used `Sequence[int]` for many places(like `spatial_size`, etc.) and also used `Tuple[int, ...]` for many places.
Should we unify them in some way?
I suggest:
1. For arg with fixed size list, use `Tuple[]`, for example: `offset: Union[Tuple[int, int], int]` in `RandShiftIntensity` transform.
2. For arg with dynamic size list, use `Sequence[int]`, for example: `spatial_size: Union[Sequence[int], int]` which can be 2D or 3D in `Resize` transform or here `in_shape`.

What do you think?
Or we can always use `Sequence[int]` as these args can accept both `list` and `tuple`, maybe `Tuple[]` is not that accurate?
Any comments @ericspod @wyli ?

Thanks.

Reply:
I would think we could use `Sequence[int]` or equivalent everywhere, there's no need to differentiate between any sequence and tuple. Wherever we'd need a tuple we should be converting inputs to tuple explicitly so that we have a local copy of the structure. For example, if `kernel_size` is a single integer or any sequence type, we'd convert it to a tuple with `ensure_tuple_rep` to get a tuple of size values that's the appropriate dimensions for the kernel in question. We'd have to do this regardless of what sequence type was passed in, even in the cases where a fixed size set of values is expected we'd likely be doing something like this and creating a new tuple from the argument value. Constructor arguments which are mutable data structures should be copied anyway to prevent clients from modifying them after construction and potentially invalidating internal state.

Reply:
Hi @benjamin-gorman , according to the above discussion, could you please help update all the `Tuple[int, ...]` to `Sequence[int]`? For `Tuple[int, int]`, I feel you can keep it to highlight the input data size for users.

Thanks.

Reply:
Converted ``Tuple[type, ...]`` to ``Sequence[type]`` and left the fixed size Tuples.